src-docs/src/views/guidelines/writing.js

```diff
@@ -233,6 +233,19 @@ export default {
               </GuideRuleWriting>
             </GuideRuleExample>
           </GuideRule>
+
+          <GuideRule heading="" description="">
+            <GuideRuleExample
+              type="do"
+              text="Do. Avoid unneeded words in button labels.">
+              <EuiButton>Create component template</EuiButton>
```

Comment:
Oh also, they're a little hard to see with the new background fill on these example panels. Let's make them a `fill` so the stand out better.
```suggestion
              <EuiButton fill>Create component template</EuiButton>
```